app/src/features/surveys/view/SurveyObservations.tsx

```diff
@@ -31,6 +71,80 @@ const SurveyObservations = () => {
     };
   };
 
+  const [submissionStatus, setSubmissionStatus] = useState<any>(null);
+  const [timer, setTimer] = useState<any>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isValidating, setIsValidating] = useState(false);
+
+  const dialogContext = useContext(DialogContext);
+
+  useEffect(() => {
+    const fetchObservationSubmission = async () => {
+      try {
+        const submission = await biohubApi.observation.getObservationSubmission(projectId, surveyId);
+
+        setSubmissionStatus(() => {
+          setIsLoading(false);
+          if (submission) {
+            if (submission.status === 'Rejected' || submission.status === 'Darwin Core Validated') {
+              setIsValidating(false);
+
+              if (timer) {
+                clearInterval(timer);
+                setTimer(null);
+              }
+            } else {
+              setIsValidating(true);
+            }
+          }
+
+          return submission;
+        });
+      } catch (e) {
+        console.error('Failed to call the API - ', e);
```

Comment:
change this so it isnt just a console error and actually throws or does something